.github/workflows/test-and-deploy.yml

```diff
@@ -33,6 +35,10 @@ jobs:
         with:
           node-version: ${{ matrix.node }}
 
+      - name: Reset env vars
+        if: env.version == 'lts/*'
```

Comment:
Can just change the matrix to use `lts` instead of `lts/*`.

Reply:
That doesn't work